apps/new/widget/home/Build.jsx

```diff
@@ -0,0 +1,353 @@
+          stroke="white"
+          stroke-width="1.5"
+          stroke-linecap="round"
+          stroke-linejoin="round"
+        />
+      </g>
+      <defs>
+        <clipPath id="clip0_80_7834">
+          <rect width="16" height="16" fill="white" />
+        </clipPath>
+      </defs>
+    </svg>
+  );
+};
+
+const Container = styled.div`
+  background: black;
+  width: 100%;
+  padding: 48px 72px;
+  display: grid;
+  grid-template-columns: repeat(2, minmax(0, 1fr));
+  align-items: center;
+  gap: 1rem;
+  @media screen and (max-width: 768px) {
+    grid-template-columns: 1fr; /* Stack in a single column */
+  }
+  @media screen and (max-width: 1024px) {
+    grid-template-columns: 1fr; /* Stack in a single column */
+    padding: 48px;
+    align-items: center;
+  }
+`;
+
+const BuildContainer = styled.div`
+  display: flex;
+  flex-direction: column;
+  gap: 24px;
+  flex-shrink: 0;
+  width: 100%;
+
+  h2 {
+    color: var(--FFFFFF, #fff);
+    font-family: Poppins;
+    font-size: 58px;
+    font-weight: 500;
+    line-height: 120%; /* 86.4px */
+    letter-spacing: -2.88px;
+    margin: 0;
+
+    span {
+      background: linear-gradient(87deg, #eca227 39.02%, #fc8119 108.97%);
+      background-clip: text;
+      -webkit-background-clip: text;
+      -webkit-text-fill-color: transparent;
+      font-family: Poppins;
+      font-size: 58px;
+      font-weight: 500;
+      line-height: 120%;
+      letter-spacing: -2.88px;
+      margin: 0;
+    }
+  }
+
+  p {
+    max-width: 421px;
+    color: var(--Color-Neutral-neutral, #666);
+    font-family: Poppins;
+    font-size: 18px;
+    margin: 0;
+  }
+
+  @media screen and (max-width: 1024px) {
+    align-items: center;
+  }
+  @media screen and (max-width: 768px) {
+    align-items: center;
+    h2 {
+      font-size: 52px;
+      span {
+        font-size: 52px;
+      }
+    }
+    p {
+      font-size: 16px;
+    }
+  }
+  @media screen and (max-width: 500px) {
+    h2 {
+      font-size: 32px;
+      span {
+        font-size: 32px;
+      }
+    }
+  }
+`;
+
+const Cards = styled.div`
+  display: grid;
+  grid-template-columns: repeat(2, minmax(0, 1fr));
+  gap: 34px;
+
+  @media screen and (max-width: 500px) {
+    grid-template-columns: repeat(2, minmax(0, 1fr));
+    gap: 16px;
+  }
+`;
+
+const BosContainer = styled.div`
+  border-radius: 24px;
+  grid-column: span 2 / span 2;
+  background: var(--1E1E1E, #1e1e1e);
```

Comment:
I really don't understand the naming of these color var
I will have to go through the UI to figure which color is meant for what
Let's create a new ticket to improve this